nautobot_device_lifecycle_mgmt/filters.py

```diff
@@ -958,12 +958,12 @@ def search(self, queryset, name, value):  # pylint: disable=unused-argument
 class VulnerabilityLCMFilterSet(NautobotFilterSet, StatusModelFilterSetMixin):  # , CustomFieldModelFilterSet):
     """Filter for VulnerabilityLCMFilterSet."""
 
-    q = django_filters.CharFilter(method="search", label="Search")
+    q = django_filters.CharFilter(method="search", label="Search")  # pylint: disable=nb-no-char-filter-q
 
-    cve__published_date = django_filters.DateTimeFromToRangeFilter()
-    cve__published_date__gte = django_filters.DateFilter(field_name="cve__published_date", lookup_expr="gte")
-    cve__published_date__lte = django_filters.DateFilter(field_name="cve__published_date", lookup_expr="lte")
-    cve__severity = django_filters.ChoiceFilter(field_name="cve__severity", choices=CVESeverityChoices)
+    cve__published_date = django_filters.DateTimeFromToRangeFilter()  # pylint: disable=nb-warn-dunder-filter-field
```

Comment:
This one, and the cve__severity are valid dunder lookups, and the disable can be added. I don't believe the others are valid though.